spec/strategy_spec.rb

```diff
@@ -34,7 +34,7 @@ def sign_into_cas(username, password)
 
     it 'should redirect to sign-in' do
       expect(response).to be_redirect
-      expect(response).to redirect_to(new_user_session_url)
+      expect(response).to redirect_to(new_user_session_url, allow_other_host: true)
```

Comment:
This one is actually the RSpec `redirect_to` matcher, rather than the ActionDispatch `redirect_to` method.  It doesn't take `allow_other_host: true`, so we should remove it here.

Reply:
Perfect, I have removed it.